server/channels/app/user.go

```diff
@@ -933,6 +933,10 @@ func (a *App) userDeactivated(c request.CTX, userID string) *model.AppError {
 		a.disableUserBots(c, userID)
 	}
 
+	if nErr := a.Srv().Store().OAuth().RemoveAuthDataByUserId(userID); nErr != nil {
```

Comment:
Should an extra test be added in the `UpdateUserActive` method's tests to cover that branch?